src/components/loading/loading.ts

```diff
@@ -189,6 +190,7 @@ class LoadingCmp {
     }
 
     this.id = (++loadingIds);
+    this.enabled = false;
```

Comment:
Maybe, I'm not entirely sure.  I updated it since it was already there.  If there is a background dismiss option or anything like that, then yes.  I don't think that makes much sense in the context of a loading indicator, so probably not.  I will remove and we can add back if needed.